home/templates/home/home_page.html

```diff
@@ -1,11 +1,41 @@
 {% extends "base.html" %}
 {% load static %}
 {% load wagtailcore_tags %}
-
+ 
 {% block content %}
 
 <h1>A special template for the front page</h1>
 <p>
+<div class="row">
+  <div class="col-sm-5">
```

Comment:
This method will work, however using `col-sm-5` for 3 columns will sum to 15, which overflows. If you read in the grid documentation, you find the following:

>  There are 12 template columns available per row, allowing you to create different combinations of elements that span any number of columns. Column classes indicate the number of template columns to span (e.g., col-4 spans four). widths are set in percentages so you always have the same relative sizing.

https://getbootstrap.com/docs/5.1/layout/grid/

Using the class `col-sm-4` would work, as 3*4 = 12.